docs/swupd.1.rst

```diff
@@ -433,7 +433,7 @@ SUBCOMMANDS
         available on the version url server, and what version number is
         available.
 
-    - `-d, --download`
+    - `-D, --download`
```

Comment:
This option isn't going to be used very frequent, so I would remove the short option

Reply:
sounds good